### PR TITLE
[WIP]Laser Musket: A makeshift low-power disabler

### PR DESCRIPTION
**Pull request by Tacolizard**
This PR adds the Laser Musket, a craftable disabler with a small cell capacity of only three shots, but using a modified disabler beam that will bring someone down in three hits. This means every hit must land on target. The musket is a bulky item that fits on the back, like the laser cannon.

todo:
sprites
~~make it craftable~~
~~fix it so it actually fires~~

**Sprites would be appreciated**

:cl: Spess Wars: Return of The Taco
add: added a craftable Laser Musket that fires disabler shots.
/:cl:

~~[why]:I feel like the game really needs more makeshift weapons, as the only way to get weapons currently is to bug sci or rob the armory. I think the laser musket is a good compromise between power and convenience.~~

**why:**
meme

### CODE REVIEW

**Comment by ShadowDeath6** [comment]
Can you swap out the cell for another one or does it work like every other energy weapon

**Comment by Tacolizard** [comment]
@ShadowDeath6 
it's like every other energy weapon, the cell is internal and inaccessible.

**Comment by Niknakflak** [comment]
A SETTLEMENT NEEDS YOUR HELP

**Comment by ShadowDeath6** [comment]
initial commit :thinking:

**Comment by Tacolizard** [comment]
@ShadowDeath6 🤔

**Comment by Tacolizard** [comment]
@optimumtact 

I think you're trying to point something out, but honestly what is wrong with that image?

**Comment by sawrge** [comment]
There doesn't need to be more fucking weapons.

**Comment by Tacolizard** [comment]
@sawrge

meme

**Comment by Tacolizard** [comment]
ok so i can spawn it ingame but for some reason it can't charge. I have no clue why

**Comment by ShadowDeath6** [comment]
why do you @ people to reply when they're already notified of the thread

**Comment by Tacolizard** [comment]
@ShadowDeath6 uh

**Comment by Anonmare** [comment]
>tfw someone steals your idea
gib royalty shekels now goyim

**Comment by kevinz000** [comment]
daily reminder that stamina damage is fucking lethal and even a single hit from a disabler can be absolutely deadly in combat :-1: if you want improvised weapons do ones that do straight damage, not crappy stuns, we do not need more stun (or worse stamina which is stun but you're stunlocked the moment you GET stunned for good!) weapons in the game.

File: code/modules/crafting/recipes.dm [hunk 1]
```diff
@@ -262,6 +262,19 @@
...
 	category = CAT_WEAPON
 
+/datum/crafting_recipe/lmusket
+	name = "Laser Musket"
+	result = /obj/item/weapon/gun/energy/l_musket
+	reqs = list(/obj/item/weaponcrafting/reciever = 1,
+				/obj/item/pipe = 1,
+				/obj/item/stack/cable_coil = 1,
+				/obj/item/stack/sheet/plasteel = 1,
+				/obj/item/weapon/shard = 1)
```
Comment (GunHog) [inline code/modules/crafting/recipes.dm]: Where are the electrical components? Could it not require a microlaser (autolathe) or something?

**Comment by Haevacht** [comment]
This is fucking pathetic.

**Comment by BeeSting12** [comment]
kevinz doesnt a single hit from a disabler do nothing? takes two for noticeable slowdown i thought

**Comment by ShadowDeath6** [comment]
48 stam damage is pretty noticeable since it basically means if you have half health you're fucked

literally does 99 stam damage in 2 hits so you can throw a glass shard at them to permafuck them

**Comment by kevinz000** [comment]
WHAT THE HELL? 48? OH HELL NO, THAT SLOWS IN A SINGLE HIT. TWO HITS YOU'RE CRAWLING. ONE HIT IF YOU HAVE ANY DAMAGE AT ALL AND YOU'RE CRAWLING. THAT'S EVEN WORSE @BeeSting12 . He fucking buffed it to slow on single hit and slow you to a CRAWL if you're damaged at all or get hit twice. Stamina damage is how you stunlock people, this is a shit idea and a shit weapon.

**Comment by Haevacht** [comment]
This is a terrible idea.

**Comment by Supermichael777** [comment]
stamina damage is different for some reason

File: code/modules/crafting/recipes.dm (hunk 1 above)
Comment (Supermichael777) [inline code/modules/crafting/recipes.dm]: pretty sure its just one bit of wire too

File: code/modules/projectiles/projectile/beams.dm [hunk 2]
```diff
@@ -62,6 +62,17 @@
 	impact_effect_type = /obj/effect/overlay/temp/impact_effect/blue_laser
 	light_color = LIGHT_COLOR_BLUE
 
+/obj/item/projectile/beam/m_disabler
+	name = "musket beam"
+	icon_state = "ice_2"
+	damage = 48 //three shots to down someone
```
Comment (Supermichael777) [inline code/modules/projectiles/projectile/beams.dm]: are you high this is 2 shots and a punch

Comment (GunHog) [inline code/modules/projectiles/projectile/beams.dm]: A piece of trash you piece together should not be superior to the proper ones in any manner.

**Comment by GunHog** [comment]
Make the weapon require winding up for 2-3 seconds per shot if you are keeping the power.

**Comment by Bawhoppen** [comment]
Laser musket should fire one time before you have to use the bluespace ramrod

**Comment by Tacolizard** [comment]
@kevinz000 actually no, it takes three hits to knock someone over. Also I do get your sentiment about having makeshift weapons deal straight non-stamina damage, but in this game nobody will use a weapon that isn't lethal. That means if I were to make this deal brute and be lethal people would scream about balance. If i made it nonlethal, people would just go back to using spears.

**Comment by Tacolizard** [comment]
@GunHog this was my original idea, I'm just not very good at coding so It would take me a while to figure out how to do it

**Comment by Tacolizard** [comment]
I am open to balance suggestions, and I agree it should do less stamina damage.

**Comment by optimumtact** [comment]
I suggest you close the pr

File: code/modules/crafting/recipes.dm (hunk 1 above)
Comment (Tacolizard) [inline code/modules/crafting/recipes.dm]: that's a good idea. I'll add that.

**Comment by Supermichael777** [comment]
Considering the modern security detention method is to simply beat cuffed prisoners for the length of their sentence I don't see the harm in greycurity.

**Comment by Bawhoppen** [comment]
No really take my suggestion

**Comment by kevinz000** [comment]
@Tacolizard I don't think you understand how STAMINA DAMAGE WORKS. Please stop adding weapons utilizing it until you fully understand how it works and what implications happens when people start taking stamina damage. It does not take "three hits to knock someone over", the first hit is a death sentence with your 48 stamdamage. Anything less is still too powerful, any more would be rediculous.

**Comment by kevinz000** [comment]
Also, lowering it by a mere 2 is not going to help. I still do not think you understand how certain combat mechanics work and even my non-meme opinion is for you to stop.

**Comment by ShadowDeath6** [comment]
remember that if your current damage + stamina damage exceeds 100 you drop

**Comment by sawrge** [comment]
People don't need more makeshift weapons. Normal crew don't need easy access to guns.  Normal crew don't need easy access to guns.  Normal crew don't need easy access to guns.  Normal crew don't need easy access to guns. 
 Normal crew don't need easy access to guns. 
 Normal crew don't need easy access to guns. 
 Normal crew don't need easy access to guns.

**Comment by Iamgoofball** [comment]
what if we removed all the modular recievers and modular memes and replaced it with a proper framework for crafting any type of gun

maybe we can call it 

# guncrafting

**Comment by Bawhoppen** [comment]
@Iamgoofball only if its not incomplete and completely unbalanced

**Comment by Iamgoofball** [comment]
remember the framework worked it was just unbalanced

**Comment by Tacolizard** [comment]
i hate you weenies

**Comment by ShadowDeath6** [comment]
Ok guys next time make a disabler that has one shot but does exactly 100 stam damage >:^)

**Pull request closed without merging**